examples/rand.py

```diff
@@ -0,0 +1,22 @@
+# various RNGs Python example
+
+import oqs
+
+#######################################################################
+# randomness example
+#######################################################################
+
+entropy_seed = [0] * 48
+entropy_seed[0] = 100
+entropy_seed[20] = 200
+entropy_seed[47] = 150
```

Comment:
I don't understand lines 9-12? Why these values? Perhaps add some comments before each wrapper call to explain the purpose of each function.

Reply:
@christianpaquin Thanks, will fix! Those are just random numbers so we have a non-zero seed, will comment in the code.